app/models/service_template_transformation_plan.rb

```diff
@@ -76,6 +76,10 @@ def self.create_catalog_item(options, _auth_user = nil)
   end
 
   def update_catalog_item(options, _auth_user = nil)
+    if options[:config_info].present? && options[:config_info][:warm_migration_cutover_datetime].present?
```

Comment:
I think you can just use `Hash#dig` here, e.g. `if options.dig(:config_info, :warm_migration_cutover_datetime)`.

Reply:
@djberg96 Sure, fixed. Thanks.